lms/static/js/staff_debug_actions.js

```diff
@@ -127,4 +136,8 @@ $(document).ready(function() {
         StaffDebug.rescore($(this).parent().data('location-name'), $(this).parent().data('location'));
         return false;
     });
+    $courseContent.on("click", '.staff-debug', function() {
+        StaffDebug.check_sudo_and_view_modal($(this).data('element-id'), $(this).data('is-sudo'));
+        return false;
+    });
```

Comment:
@cpennington This is for staff-debug-info, but it doesn't mean that if user change its value manually and then he can do staff user stuff it will give them a error on doing that, it is only for showing the modal.